pkg/provisioner/ironic/ironic.go

```diff
@@ -673,16 +673,18 @@ func (p *ironicProvisioner) InspectHardware(force bool) (result provisioner.Resu
 				p.log.Info("inspection already started")
 				result, err = operationContinuing(introspectionRequeueDelay)
 				return
-			default:
-				if nodes.ProvisionState(ironicNode.ProvisionState) == nodes.InspectFail && !force {
+			case nodes.InspectFail:
+				if !force {
```

Comment:
How about this instead?

```
if force {
  fallthrough
}

to avoid nested if

```

Reply:
That's not allowed (`fallthrough` has to be the last statement).

Reply:
hmm, yea you are right, `fallthrough statement out of place`